lib/App/DH.pm

```diff
@@ -281,6 +317,8 @@ my $list_cmds_usage =
     cmd_write_ddl
     cmd_install
     cmd_upgrade
+    cnd_database_version
```

Comment:
I suspect this should be "cmd" not "cnd"

Also, ideally these commits should be squashed down to a single commit, transient states without inherent value that never make it to a target aren't entirely useful and can be avoided :)